src/app/shared/dialogs/dialogs-list.service.ts

```diff
@@ -13,11 +14,26 @@ const listColumns = {
 @Injectable()
 export class DialogsListService {
 
+  defaultSelectors = {
```

Comment:
I'm not sure if we want defaultselector of user here. As this dialoglist is used for resources and courses as well.